src/config/graphical-session/graphics-drivers/index.md

```diff
@@ -7,3 +7,4 @@ of your system.
 
 - [Intel GPU](./intel.md)
 - [NVIDIA Optimus](./optimus.md)
+- [NVIDIA](./nvidia.md)
```

Comment:
This should be "Nvidia". (Leave the "NVIDIA Optimus" link text as it is for now; `optimus.md` needs work anyway, so that change can go in the relevant PR.)